MobileCenterCrashes/MobileCenterCrashes/MSCrashes.h

```diff
@@ -95,37 +95,23 @@ typedef NS_ENUM(NSUInteger, MSUserConfirmation) {
 ///-----------------------------------------------------------------------------
 
 /**
- *  Trap fatal signals via a Mach exception server.
+ * Disable to trap fatal signals via a Mach exception server.
```

Comment:
This is a merge PR and your feedback needs to be addressed separately. #642 will do that. Thanks!